incubator-tools/cmek_docai_processor/cmek_docai_processor.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #40.            update_mask = {'paths': ['rotation_period', 'next_rotation_time']}
</pre><p>Use the built in class instead of a dict when possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Documentation has the same thing which we used . <a href="https://cloud.google.com/kms/docs/rotate-key#kms-create-key-rotation-schedule-python" target="_blank">https://cloud.google.com/kms/docs/rotate-key#kms-create-key-rotation-schedule-python</a></p>